Calibration/PPSAlCaRecoProducer/test/2022_lhcinfo_test_AlCaRecoProducer.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+function die { echo $1: status $2; exit $2; }
+
+customise_commands="process.GlobalTag.toGet = cms.VPSet()\
```

Comment:
these should be taken from the DB (after Wagner uploads them), and I guess not just the DB but through a GT

Reply:
this test was temporary and was removed in https://github.com/cms-sw/cmssw/pull/42515/commits/0d1170c5b303f20b1b06568b994a27f55a1d017b since 2023_lhcinfo_test_recoCTPPS_cfg.py performs a similar task. The afformentioned commit will be squashed once the pr is tested